ext/storage_sources/azure_store/azure_connection.cpp

```diff
@@ -178,3 +207,14 @@ azure_connection::bucket_exists(bool &exists) const
     }
     return 0;
 }
+
+// Maps the HTTP code returned by Azure SDK and returns the associated errno code if it is in
```

Comment:
We can break up this run-on sentence into two:
```
// Return a system error code corresponding to the HTTP code returned by
// the Azure SDK. If no such mapping is defined, return -1.
```

Reply:
resolved here https://github.com/wiredtiger/wiredtiger/pull/8750/commits/e42f214727eda92b86bcea0d45fc6c00676173e8